README.md

```diff
@@ -374,40 +111,28 @@ There can be issues running the docker daemon from _inside the dev container_ (a
 
 ## Contributing
 
-There are a couple of ways you can contribute to this repository:
-
-* Ideas, feature requests and bugs: We are open to all ideas and we want to get rid of bugs! Use the Issues section to either report a new issue, provide your ideas or contribute to existing threads
-* Documentation: Found a typo or strangely worded sentences? Submit a PR!
-* Code: Contribute bug fixes, features or design changes.
-
-## Legal
-
-Before we can accept your pull request you will need to sign a **Contribution License Agreement**. All you need to do is to submit a pull request, then the PR will get appropriately labelled (for example `cla-required`, `cla-norequired`, `cla-signed`, `cla-already-signed`). If you already signed the agreement we will continue with reviewing the PR, otherwise system will tell you how you can sign the CLA. Once you sign the CLA all future PR's will be labeled as `cla-signed`.
+There are a couple of ways you can contribute to this repo:
 
-## Troubleshooting
+* **Ideas, feature requests and bugs**: We are open to all ideas and we want to get rid of bugs! Use the Issues section to either report a new issue, provide your ideas or contribute to existing threads.
+* **Documentation**: Found a typo or strangely worded sentences? Submit a PR!
+* **Code**: Contribute bug fixes, features or design changes:
```

Comment:
nit: fixes, features, or design changes

Reply:
Lol fair enough but all of our repos are like this

Reply:
> **Documentation**: Found a typo or strangely worded sentences? Submit a PR!

Guess I'll submit a bunch of PRs then.